app/routes/features-list.js

```diff
@@ -1 +0,0 @@
-export { default } from 'ember-feature-controls/routes/features-list'
```

Comment:
I'm not familiar to typescript addons, but why we don't need to export things anymore on `app/` ?

Reply:
I deleted this route not due to TS, but for these reasons:

1. In `router.ts`, the `features-list` route is declared
   1. https://github.com/DazzlingFugu/ember-feature-controls/blob/upgrade-to-emberjs-5.3.0/tests/dummy/app/router.ts
   1. https://github.com/DazzlingFugu/ember-feature-controls/blob/upgrade-to-emberjs-5.3.0/addon/route-setup.ts
1. `addon/routes/features-list.js` is a blank Route + because it is declared in the Route => it can be deleted
1. Following (2), `app/routes/features-list.js` can also be deleted.
1. Following (2), `tests/unit/routes/features-list-test.js` can also be deleted.

Reply:
Yup just tested and you are right, thanks for clarification!